ansible/README.md

```diff
@@ -0,0 +1,34 @@
+# ansible-microshift
+
+The purpose of this ansible automation is to help gain insight into the footprint and start times of MicroShift.
+At this stage we are capturing the first start as well as the second start (with cached images), as well as disk
+usage data from a number of points in time as well as performance tool data with pbench for each of the two starts.
+
+## Prereqs
+
+We are using [pbench](https://github.com/distributed-system-analysis/pbench) to capture a wide array of performance tool data.
+Pbench has Ansible playbooks available in Ansible Galaxy to install the pbench-agent on the MicroShift host.
+
+- Ansible
+  - `ansible-galaxy collection install pbench.agent`
```

Comment:
Need instructions on installing ansible-galaxy as it's not available in standard repos.

Reply:
Added Ansible installation doc step for Fedora.